core/templates/domain/learner_group/learner-group-user-progress.model.ts

```diff
@@ -22,6 +22,12 @@ import { LearnerGroupSubtopicSummary, LearnerGroupSubtopicSummaryBackendDict }
  * user progress domain objects.
  */
 
+export interface LearnerGroupInvitedUserInfo {
```

Comment:
Is this a backend dict (since I see camel case)? If so, can you rename it to include that?

Reply:
moved this to a separate model file as per our discussion